validators/validation.js

```diff
@@ -0,0 +1,16 @@
+const Joi = require("joi");
+
+exports.signUpSchema = Joi.object({
+  email: Joi.string().email().lowercase().required(),
+  password: Joi.string().min(4).max(15).required().label("Password"),
+  role: Joi.string().required(),
+  confirmPassword: Joi.any()
```

Comment:
Nice use of Joi. However, when building API, you do not need confirm password again. Confirm password is only for frontend to handle. You should only have one password field.

In addition, see how you can add other validation constraint like one number or special character required